internal/common/baseclient/basejudge.go

```diff
@@ -64,6 +64,10 @@ func (j *BaseJudge) InspectHistory(iigoHistory []shared.Accountability) (map[sha
 	return outputMap, true
 }
 
+func (j *BaseJudge) GetPardonedIslands(currentSanctions map[int][]roles.Sanction) map[int][]bool {
```

Comment:
clientID and not int?
also a comment woould be great as to the usage of this function

Reply:
The int's mentioned here in the map are representative of how long ago the sanction was levied. Admittedly an array would've done the trick but the map seemed convenient at the time.